src/shared/config.ts

```diff
@@ -77,48 +78,19 @@ export interface ServiceConfig {
   }
   SHARED: {
     PEER_ENDPOINT: string
-    ALS_ENDPOINT: string
-    QUOTES_ENDPOINT: string
-    TRANSFERS_ENDPOINT: string
-    BULK_TRANSFERS_ENDPOINT: string
+    ALS_ENDPOINT?: string
+    QUOTES_ENDPOINT?: string
+    TRANSFERS_ENDPOINT?: string
+    BULK_TRANSFERS_ENDPOINT?: string
+    THIRDPARTY_REQUESTS_ENDPOINT?: string
```

Comment:
It seems to be forgot to add `THIRDPARTY_REQUESTS_ENDPOINT` and `TRANSACTION_REQUEST_ENDPOINT` in config files

Reply:
there are optional parameters, if not specified PEER_ENDPOINT will be used